### PR TITLE
ties in RNA branch to sql-based repo manager

### LINKED ISSUE #1227: RNA data needs to be managed by repomanager
Update RNA topic branch to:
- remove references to prior data directory structure
- add functions to manage RNA data to `cli.py` and probably also `datarepo.py`

### PR DESCRIPTION
**Pull request by saupchurch**
fixes #1227

### CODE REVIEW

File: ga4gh/datamodel/rna_quantification.py [hunk 1]
```diff
@@ -129,6 +129,7 @@ def __init__(self, parentContainer, localId):
         self._description = ""
         self._name = localId
         self._readGroupId = ""
+        self._referenceSet = ""
```
Comment (jeromekelleher) [inline ga4gh/datamodel/rna_quantification.py]: Should probably default to None.

**Comment by jeromekelleher** [comment]
Looks great to me @saupchurch! +1

**Comment by saupchurch** [comment]
thanks @jeromekelleher change made

**Comment by saupchurch** [comment]
per integration group call - removed WIP decorator.
Ready to review for merge

**Comment by macieksmuga** [comment]
+0.9 Don't merge quite yet...

The last +0.1 is waiting on a minor update that Sean and I discussed.

**Pull request merged**